broadinstitute/orsp-pub
# ORSP Portal Update - Implement new status/button for Submit to IRB

## Addresses
https://broadinstitute.atlassian.net/browse/CRIC-1260

## Changes
ORSP Portal Update - Implement new status/button for Submit to IRB

## Testing
1) Add a "Submitted to IRB" button to the left of "Reject" 
2) Change the default "Pending" status on the right (in yellow) to "Pending ORSP Review" when someone submits a sample/data cohort 
3) Add a new status (in orange/different color), "Pending IRB Review," when ORSP hits the new "Submitted to IRB" button on the left. 
4) Update the email text that users receive when they submit a sample/data cohort. Change current text "A new Consent Group Record xxxx has been created in the ORSP Portal and is pending ORSP review and approval." to new text: "A new Sample/Data Cohort Record xxxx has been created in the ORSP Portal and is pending ORSP review and approval. Please note that work with the samples or data should not start until the sample/data cohort has been approved by ORSP and/or the IRB-of-record (as needed). Approval status is shown in the sample/data cohort tab."

---
